### PR TITLE
Add temporary information about readonly file system causing woes

### PR DESCRIPTION
**Pull request by WillGibson**

### CODE REVIEW

File: commands/templates/svc/manifest-public.yml [hunk 1]
```diff
@@ -66,6 +66,15 @@ network:
   vpc:
     placement: 'private'
 
+# The application currently does not work of the file system is readonly, e.g...
```
Comment (yusufsheiqh) [inline commands/templates/svc/manifest-public.yml:69]:
```suggestion
# The application currently does not work if the file system is readonly, e.g...
```

File: commands/templates/svc/manifest-public.yml [hunk 2]
```diff
@@ -66,6 +66,15 @@ network:
...
+#
+# File "/layers/paketo-buildpacks_cpython/cpython/lib/python3.9/tempfile.py", line 291, in gettempdir
+# tempdir = _get_default_tempdir()
+# File "/layers/paketo-buildpacks_cpython/cpython/lib/python3.9/tempfile.py", line 223, in _get_default_tempdir
+# raise FileNotFoundError(_errno.ENOENT
+# FileNotFoundError: [Errno 2] No usable temporary directory found in ['/tmp', '/var/tmp', '/usr/tmp', '/workspace']
```
Comment (yusufsheiqh) [inline commands/templates/svc/manifest-public.yml:75]: I am actually not sure we need all this. A sentence or two stating why the following two lines need commenting out would be sufficient, IMO.

Comment (WillGibson) [reply commands/templates/svc/manifest-public.yml]: How about now?

Comment (yusufsheiqh) [reply commands/templates/svc/manifest-public.yml]: Yup, that's nice and smol.

Comment (yusufsheiqh) [reply commands/templates/svc/manifest-public.yml]: Resolved in https://github.com/uktrade/copilot-tools/pull/157/commits/4f781d00eeffc32e0ef67bb0b4b6da00437f1d4e.

**Review by yusufsheiqh — COMMENTED** [review]
Tests are failing ATM, and the package version will also need to be bumped up.

**Review by yusufsheiqh — APPROVED** [review]

**Pull request merged**